driver/src/main/java/org/neo4j/driver/internal/async/pool/NettyChannelPool.java

```diff
@@ -236,7 +236,7 @@ public String id() {
         return this.id;
     }
 
-    private String poolId(BoltServerAddress serverAddress) {
-        return String.format("%s:%d-%d", serverAddress.host(), serverAddress.port(), this.hashCode());
+    private String poolId(SocketAddress serverAddress) {
+        return String.format("pool-%d-%d", serverAddress.hashCode(), this.hashCode());
```

Comment:
Not sure if we should change this as it is exposed via: `ConnectionPoolMetrics#id()`.